CometServer/Entities/Universe.cpp

```diff
@@ -167,18 +167,19 @@ namespace entity
 				{
 					if (p_entity1 == p_entity2) continue;
 					//Test for collision.
-					#ifndef NO_COLLISION
+					/*#ifndef NO_COLLISION
```

Comment:
Unfortunately this is in a distinct compilation unit from the `GameServer`'s so there's no convenient way to define this macro, except for creating a new build target that automatically supplies it to every compilation unit. That's what we are going to do in the next pull request.